sass/elements/_schedule.scss

```diff
@@ -0,0 +1,163 @@
+$white: #ffffff;
+$black: #000000;
+$shadow: rgba(0, 0, 0, 0.5);
+
+@mixin schedule-block($bg-color, $txt-color) {
+  border-bottom: $bg-color solid 10px;
+  background: $white;
+  padding: 5px;
+  margin: 5px;
+
+  a {
+    color: $black;
+  }
+
+  &:hover {
+    background-color: tint($bg-color, 60%);
+  }
+
+  box-shadow: 0 2px 4px $shadow;
+}
+
+.tables-container {
+  width: 100%;
+  text-align: center;
+
+  .row-headers {
+    width: 8%;
+  }
+}
+
+.schedule-table {
+  border-collapse: separate;
+  table-layout: fixed;
+  border-spacing: 2px;
+  display: inline-block;
+  vertical-align: top;
+  text-align: left;
+  width: 11%;
+
+  thead {
+    th {
+      border: 0;
+    }
+  }
+
+  tr {
+    height: 84px;
+  }
+
+  td,
+  th {
+    border-top: 0;
+    overflow-y: hidden;
+
+    p {
+      display: block;
+      width: 100%;
+      height: 30px;
+    }
+  }
+
+  .time-header {
+    border: 0;
+    text-align: center;
+
+    .mobile-only {
+      display: none;
+    }
+  }
+
+  .col-headers {
+    height: 24px;
+  }
+
+  .schedule-timeslot:not(.schedule-blocks-sustainer) {
+    cursor: pointer;
+
+    &:hover {
+      text-decoration: underline;
+    }
+  }
+}
+
+$sustainer-color: tint($primary-color, 40%);
+
+.schedule-blocks-sustainer {
+  @include schedule-block($sustainer-color, $black);
+  background-color: $sustainer-color;
+
+  &:hover {
+    background-color: $sustainer-color;
+  }
+}
+
+.schedule-blocks-regular {
+  @include schedule-block($primary-color, $off-white-color);
+}
+
+.schedule-blocks-flagship {
+  @include schedule-block($flagship, $black);
+}
+
+.schedule-blocks-news {
+  @include schedule-block($news-maroon, $off-white-color);
+}
+
+.schedule-blocks-speech {
+  @include schedule-block($speech-green, $off-white-color);
+}
+
+.schedule-blocks-music {
+  @include schedule-block($music-purple, $off-white-color);
+}
+
+.schedule-blocks-event {
+  @include schedule-block($red, $off-white-color);
+}
+
+.schedule-blocks-specialistMusic {
+  @include schedule-block(tint($music-purple, 60%), $white);
+}
+
+.schedule-blocks-specialistInterest {
+  @include schedule-block(tint($speech-green, 60%), $white);
+}
+
+@media screen and (max-width: 767px) {
```

Comment:
Consider using the breakpoint constants here, see https://github.com/UniversityRadioYork/2016-site/blob/development/sass/base/_utils.scss#L15 for an example